lms/djangoapps/courseware/views/index.py

```diff
@@ -100,21 +100,6 @@ def get(self, request, course_id, chapter=None, section=None, position=None):
             position (unicode): position in module, eg of <sequential> module
         """
         self.course_key = CourseKey.from_string(course_id)
-
-        if not (request.user.is_authenticated or self.enable_anonymous_courseware_access):
```

Comment:
please look at this PR
https://github.com/raccoongang/edx-platform/pull/1172/files
here the access was limited.

Reply:
Changed